src/config/mod.rs

```diff
@@ -21,6 +21,11 @@ pub struct Config {
     pub addons: Addons,
 
     pub theme: Option<String>,
+
+    #[serde(default)]
+    pub column_config: ColumnConfig,
+
+    pub window_size: Option<(u32, u32)>,
```

Comment:
Great addition we actually save it to config. This is were it becomes really useful.